Workbooks/CosmosDb/Cost Optimization/Optimizing Total Provisioned Throughput.workbook

```diff
@@ -0,0 +1,1002 @@
+            "name": "text - 0"
+          }
+        ]
+      },
+      "name": "group - 7"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "<span style=\"font-size:30px;\"> Background Information </span>\r\n\r\n<br>\r\n<span style = \"font-size:15px;\"> Before we jump into analyzing our workload, there are a few concepts that are important to understand.\r\n## Billing Overview\r\n<span style = \"font-size:15px;\"> In Azure Cosmos DB, you are billed based on your database operations and storage. For database operations, users can choose between manual and autoscale for provisioning throughput. You will be billed based on how many RU/s you provision, regardless of how many you consume. Therefore, selecting the right method of provisioning throughput can help you optimize performance and costs.\r\n<br> \r\n\r\n<span style = \"font-size:15px;\">Manual vs. Autoscale:\r\n-\tManual Provisioned Throughput:  You set the maximum throughput, measured in request units per second (RU/s) required for your workload. The service provisions the capacity needed to support the throughput requirements. You are billed hourly for the maximum provisioned throughput.\r\n-\tAutoscale: You set the highest maximum throughput that you do not want to exceed. Your system automatically scales the throughput T such that 0.1* Tmax <= T <= Tmax. You are only billed for the highest amount of RU/s your system scales every hour. \r\nStorage: User is billed a flat rate for total amount of storage (in GBs) consumed by data and indexes for a given hour. \r\n<br>\r\n\r\n<span style = \"font-size:15px;\">Now that we know how billing works in Azure Cosmos DB and the different choices for provisioning throughput, let’s study how much of our provisioned throughput we are using. As mentioned earlier, you will be billed based on how many RU/s you provision, regardless of how many you consume, so it is important to right-size your workload in order to optimize costs and performance.  \r\n"
+            },
+            "customWidth": "100",
+            "name": "text - 0"
+          }
+        ]
+      },
+      "name": "group - 7 - Copy"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "<span style=\"font-size:30px;\"> Task 1: Analyze Your Utilization </span>\r\n## Select time range and container to analyze\r\n<span style = \"font-size:15px;\"> Let’s begin by selecting the time range we’d like to study from this drop-down menu. You can choose from the last 5 hours up to the last 90 days. You should also select which container you would like to analyze from the resource drop-down menu.\r\n<br> "
```

Comment:
```suggestion
              "json": "<span style=\"font-size:30px;\"> Task 1: Analyze Your Utilization </span>\r\n## Select time range and container to analyze\r\n<span style = \"font-size:15px;\"> Let’s begin by selecting the time range we’d like to investigate from this drop-down menu. You can choose from the last 5 hours up to the last 90 days. You should also select which container you would like to analyze from the resource drop-down menu.\r\n<br> "
```

Reply:
Need to have Anyone preselected in the workbook in order for me to select the db: 
<img width="140" alt="image" src="https://user-images.githubusercontent.com/29219340/196502542-0c46fb43-e0bf-4ab4-8556-0c1e88da903e.png">

Reply:
HIde the Autoscale DB parameter: 
<img width="140" alt="image" src="https://user-images.githubusercontent.com/29219340/196502688-11d2ca3e-befa-4468-a833-9e3a44ae7810.png">

Reply:
Hide the nosharedThroughput parameter: 
<img width="148" alt="image" src="https://user-images.githubusercontent.com/29219340/196502797-3aee36b8-7841-4014-bfe6-f676976a70d9.png">

Reply:
Remove default time range and set it to the last 24hrs or 7days: 
<img width="172" alt="image" src="https://user-images.githubusercontent.com/29219340/196502978-ccc338ac-9c23-4fc6-9a05-25ad077cc221.png">

Reply:
Can we make this highlighted section in green or something more bold? 
<img width="530" alt="image" src="https://user-images.githubusercontent.com/29219340/196503160-98b80edd-4073-4f74-9555-802b57bf0f01.png">